Patterns/Multi-device/Multi-device.cpp

```diff
@@ -332,7 +332,7 @@ int main(int argc, char** argv)
 
     float max_relative_error = helpers::maxError(h_C, h_Gold);
     float eps                = std::numeric_limits<float>::epsilon();
```

Comment:
I think this needs to change, there is max_relative_error = helpers::maxError(h_C, h_Gold);   If the variable is max_relative_error I think it should be calling helpers::maxRelativeError. I also think we need to switch to:

float max_relative_error = helpers::maxRelativeError(h_C, h_Gold);
float eps = std::numeric_limits<float>::epsilon();
float tolerance          = K ;

This will correspond to the checking suggested by Lee. I will set up a meeting to discuss this.